tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -43,8 +45,10 @@ Define {
 	DMIC1_ID			2
 	DMIC0_HOST_PIPELINE_ID		13
 	DMIC0_DAI_PIPELINE_ID		14
-	DMIC0_HOST_PIPELINE_SINK	'copier.host.13.1'
-	DMIC0_DAI_PIPELINE_SRC		'copier.DMIC.14.1'
+	DMIC0_HOST_PIPELINE_SINK	'copier.module.13.2'
+	DMIC0_DAI_PIPELINE_SRC		'copier.module.14.2'
+	DMIC0_DAI_COPIER		'copier.DMIC.14.1'
+	DMIC0_DAI_GAIN			'gain.14.1'
```

Comment:
same here, why do we still have Passthrough Capture on line 53. It's obsolete/irrelevant naming.

Reply:
sure